packages/styles/docs/sass.md

```diff
@@ -322,21 +327,36 @@ For more information, check out our [type](#todo) docs.
 ## Components
 
 All of the styles for the components in the Carbon Design System live in the
-`scss/components` directory. You can include all of the styles for a component
-by including its entrypoint file. For a full list of component styles that you
-can import, check out the files table below.
+`scss/components` directory.
+
+**Reminder: Module paths in `@carbon/styles` and `@carbon/react` are
+synonymous - `@carbon/react` re-exports everything from `@carbon/styles`.**
+
+You can include all of the styles for a component by including its entrypoint
+file. For a full list of component styles that you can import, check out the
+files table below.
 
 **Component tokens**
 
 In some situations, you may want to change the tokens for a specific component.
-To do so you will need to configure the module and provide the tokens you would
+To do so you will need to configure the modules and provide the tokens you would
 like to see changed. For example, if you wanted to change the component token
-`button-separator` for `button` you would do the following:
+`button-primary` for `button` you would do the following:
 
 ```scss
-@use '@carbon/styles/scss/components/button' with (
-  $button-separator: #e4e4e4
+@use '@carbon/react/scss/themes';
+@use '@carbon/react/scss/components/button/tokens' with (
+  $button-primary: (
+    fallback: #3f51b5,
+    values: (
+      (theme: themes.$white, value: #3f51b5),
+      (theme: themes.$g10, value: #3f51b5),
+      (theme: themes.$g90, value: #3f51b5),
+      (theme: themes.$g100, value: #3f51b5),
+    ),
+  )
 );
+@use '@carbon/react/scss/components/button';
```

Comment:
I found out that if you add that line for the reset, the font comes in as intended! Not sure if we wanted to add that to the snippet. Also, I thought it might be nice to have different values for the themes. 

```suggestion
@use '@carbon/react/scss/themes';
@use '@carbon/react/scss/components/button/tokens' with (
  $button-primary: (
    fallback: #3f51b5,
    values: (
      (theme: themes.$white, value: #3f51b5),
      (theme: themes.$g10, value: #d55bff),
      (theme: themes.$g90, value: #d0b2ff),
      (theme: themes.$g100, value: #cfd2ff),
    ),
  )
);

@use '@carbon/react/scss/reset';
@use '@carbon/react/scss/components/button';
```

Reply:
Also would we want to add an example the like "basic" version of this? 

```
@use '@carbon/react/scss/themes';
@use '@carbon/react/scss/components/button/tokens' with (
  $button-primary: #d55bff
);

@use '@carbon/react/scss/reset';
@use '@carbon/react/scss/components/button';
```